Logger/FileLogger.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+
+        public string FilePath { get; set; } = "";
```

Comment:
```suggestion
        private string _FilePath;
```
This should be private.